lib/libutils/ext/include/util.h

```diff
@@ -30,6 +30,13 @@
 #include <compiler.h>
 #include <stdint.h>
 
+#define SIZE_4K	0x1000
```

Comment:
All these constants should be of an unsigned type, same width as `size_t` would be a good choice.

Reply:
I'm sorry but how to address this

Reply:
We maybe something like `#define SIZE_2G	0x80000000UL`